jina/orchestrate/flow/base.py

```diff
@@ -96,7 +98,11 @@ class FlowType(type(ExitStack), type(JAMLCompatible)):
 ]
 
 
-class Flow(PostMixin, JAMLCompatible, ExitStack, metaclass=FlowType):
+class _BaseClassFlow(PostMixin, JAMLCompatible, ExitStack, metaclass=FlowType):
```

Comment:
why is this new class needed?

Reply:
I need this class here : https://github.com/jina-ai/jina/blob/b70aa02900aa3957531719e3b88a00dc25b447de/jina/orchestrate/flow/base.py#L2048

Otherwise I can't have a default `__getattribute__` function which I overload to check the if the function called is `load_config`

Reply:
u can not get it from object? just doing `super().__getattribute__(item)`? It would surprise me this is not possible

Reply:
yes ofc